server/templates/static/homepage.html

```diff
@@ -18,6 +18,7 @@
  {% set main_id = 'homepage' %}
  {% set page_id = 'page-homepage' %}
  {% set title = 'Home' %}
+ {% set brand_logo_path = "/images/google-logo-reverse.svg" %}
```

Comment:
What do you think about making this a boolean like `brand_logo_dark = true`? might leave less room for error